source/standards/source-code.html.md.erb

```diff
@@ -1,44 +1,34 @@
 ---
-title: Source code
+title: How to store source code
 expires: 2018-03-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-We follow the principles set out in the service manual for
-managing the code that we write:
+At GDS, we follow the principles set out in the Service Manual for managing the code we write by:
 
-- [use version control](https://www.gov.uk/service-manual/technology/maintaining-version-control-in-coding)
-- [make source code open](https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable)
+- [using version control](https://www.gov.uk/service-manual/technology/maintaining-version-control-in-coding)
+- [making source code open](https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable)
 
-You should keep secrets separate from source code.
+You must keep secret data such as keys or credentials closed and separate from source code. This information could allow someone to gain access to your system. You should consider encrypting the contents of your repository.
 
-GDS uses GitHub.
+## Use GitHub as your repository
 
-## GitHub
+[GDS uses GitHub as its code repository](https://gdstechnology.blog.gov.uk/2016/05/31/how-we-use-git-at-the-government-digital-service/).
 
 New repositories for GDS products and services go in the
 [alphagov](https://github.com/alphagov/) organisation on GitHub.
 
-You can use your personal GitHub account at GDS. Ask your tech
-lead or technical architect to add you to the organisation.
+You can use your personal GitHub account to access alphagov. Ask your tech lead or technical architect to add you (they will need to be on the [list of alphagov owners](https://github.com/orgs/alphagov/people?utf8=%E2%9C%93&query=%20role%3Aowner) to do so).
```

Comment:
This link doesn't work if you're not logged in as an owner. In incognito mode it will display all public members of alphagov.

Reply:
We have taken this link out of the revised guidance.